core/src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -82,13 +82,14 @@ case class TiStrategy(getOrCreateTiContext: SparkSession => TiContext)(sparkSess
   private def useStreamingProcess(): Boolean =
     sqlConf.getConfString(TiConfigConst.COPROCESS_STREAMING, "false").toLowerCase.toBoolean
 
-  private def timeZoneOffset(): Int =
-    sqlConf
-      .getConfString(
-        TiConfigConst.KV_TIMEZONE_OFFSET,
-        String.valueOf(ZonedDateTime.now.getOffset.getTotalSeconds)
-      )
-      .toInt
+  private lazy val timeZoneOffset: Int = {
+    val tz = DateTimeZone.getDefault
```

Comment:
I wonder whether this is a correct behavior for `timeZoneOffset` because `timeZoneOffset` is not a lazy val, which means it may not follow the timezone changes if user use `DateTimeZone.setDefault`.

Reply:
agree